vispy/visuals/line/line.py

```diff
@@ -340,25 +341,11 @@ def _prepare_draw(self, view):
             self.shared_program['texture2D_LUT'] = cmap.texture_lut() \
                 if (hasattr(cmap, 'texture_lut')) else None
 
-        # Do we want to use OpenGL, and can we?
-        GL = None
-        from ...app._default_app import default_app
-        if default_app is not None and \
-                default_app.backend_name != 'ipynb_webgl':
-            try:
-                import OpenGL.GL as GL
-            except Exception:  # can be other than ImportError sometimes
-                pass
-
-        # Turn on line smooth and/or line width
-        if GL:
-            if self._parent._antialias:
-                GL.glEnable(GL.GL_LINE_SMOOTH)
-            else:
-                GL.glDisable(GL.GL_LINE_SMOOTH)
+        if "webgl" not in default_app_module.default_app.backend_name:
+            self.update_gl_state(line_smooth=bool(self._parent._antialias))
```

Comment:
I honestly didn't even know the `update_` variant of this function existed. I've only ever seen `set_gl_state` used. Cool.